sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/plan/DefaultStep.java

```diff
@@ -143,16 +140,6 @@ public Status getStatus() {
         }
     }
 
-    @Override
-    public List<Element<?>> getChildren() {
-        return Collections.emptyList();
-    }
-
-    @Override
```

Comment:
@nickbp:  Steps should be interruptable.  If a step is failing to succeed it will constantly retry.  In this case and interrupted step would stop retrying.  Consider the case where many steps are being deployed in parallel but one or two are having trouble and the operator would like to stop them, she might want to interrupt individual steps.